Collect code coverage and send to Coveralls

ligurio opened the pull request:
Coveralls https://coveralls.io/github/tarantool/expirationd

Closes #85

olegrok commented on Makefile:
```diff
@@ -21,11 +23,11 @@ apidoc:
-	.rocks/bin/luatest -v
+	luatest -v --coverage
 	rm -rf ${CLEANUP_FILES}
 	INDEX_TYPE='TREE' SPACE_TYPE='vinyl' ./test.lua
 	rm -rf ${CLEANUP_FILES}
```
side not: do we still needed with rm commands? Probably cleanup logic should live inside our tests.

ligurio commented:
Updated the patches - change with option `--coverage` should be in commit related to code coverage.

Old version of patch in a first commit:
```patch
--- a/Makefile
+++ b/Makefile
@@ -15,17 +15,17 @@ all: test
 # functions like string.split(), string.endswith(), so we run
 # ldoc using tarantool.
 apidoc:
-       .rocks/bin/ldoc -c $(PROJECT_DIR)/doc/ldoc/config.ld \
+       ldoc -c $(PROJECT_DIR)/doc/ldoc/config.ld \
              -d $(PROJECT_DIR)/doc/apidoc/ expirationd.lua
 
 check: luacheck
 
 luacheck:
-       .rocks/bin/luacheck --config .luacheckrc --codes .
+       luacheck --config .luacheckrc --codes .
 
 .PHONY: test
 test:
-       .rocks/bin/luatest -v
+       luatest -v --coverage
        rm -rf ${CLEANUP_FILES}
        INDEX_TYPE='TREE' SPACE_TYPE='vinyl' ./test.lua
        rm -rf ${CLEANUP_FILES}
```

So I removed change in first commit and added to the second one.
